lib/pbench/server/api/resources/datasets_daterange.py

```diff
@@ -0,0 +1,65 @@
+from flask.json import jsonify
+from flask.wrappers import Request, Response
+from logging import Logger
+from sqlalchemy import func
+
+from pbench.server import PbenchServerConfig
+from pbench.server.api.resources import (
+    ApiBase,
+    API_OPERATION,
+    Parameter,
+    ParamType,
+    Schema,
+)
+from pbench.server.database.database import Database
+from pbench.server.database.models.datasets import Dataset
+
+
+class DatasetsDateRange(ApiBase):
+    """
+    API class to retrieve the available date range of accessible datasets.
+    """
+
+    GET_SCHEMA = Schema(
+        Parameter("owner", ParamType.USER, required=False),
+        Parameter("access", ParamType.ACCESS, required=False),
+    )
+
+    def __init__(self, config: PbenchServerConfig, logger: Logger):
+        super().__init__(
+            config,
+            logger,
+            None,
+            role=API_OPERATION.READ,
+        )
+
+    def _get(self, _, request: Request) -> Response:
+        """
+        Get the date range for which datasets are available to the client based
+        on authentication plus optional dataset owner and access criteria.
+
+        Args:
```

Comment:
Can we add a note to document why the first argument is named `_`?

Reply:
We _really_ shouldn't need to do that:  it is common practice in Python (see the end of the box [here](https://docs.python.org/3/reference/lexical_analysis.html#reserved-classes-of-identifiers)) to indicate unused parameters by naming them `_`.

(But, I see that Dave has already addressed this...so...nevermind?)